src/Text/Pandoc/Readers/Roff.hs

```diff
@@ -433,12 +433,13 @@ tableColFormat = do
                 else return ""
       return $ x : num
     pipeSuffix' <- option False $ True <$ string "|"
+    skipMany (letter <|> digit)
```

Comment:
I don't think we should skip these at this (tokenization) phase.

Reply:
Perhaps the better solution is to move line 424 (numsuffixes) to right before 434.  That is, the numerical suffix comes after all other suffixes.  I wasn't clear from the tbl manual whether it worked this way, but it seems from your examples that maybe it does...